doc/source/whatsnew/v0.18.2.txt

```diff
@@ -101,6 +101,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13418`)
```

Comment:
I get it but should do that change for 0.19 (should have done it for 0.18 but that ship sailed)